spec/models/form/sales/pages/about_price_value_check_spec.rb

```diff
@@ -27,27 +28,6 @@
     expect(page.interruption_screen?).to eq(true)
   end
 
-  it "has the correct title_text" do
-    expect(page.title_text).to eq({ "arguments" => [{ "i18n_template" => "value", "key" => "value", "label" => true }], "translation" => "soft_validations.purchase_price.title_text" })
-  end
-
-  it "has the correct informative_text" do
-    expect(page.informative_text).to eq({
-      "translation" => "soft_validations.purchase_price.hint_text",
-      "arguments" => [
-        {
-          "key" => "field_formatted_as_currency",
-          "arguments_for_key" => "purchase_price_soft_min_or_soft_max",
-          "i18n_template" => "soft_min_or_soft_max",
-        },
-        {
-          "key" => "purchase_price_higher_or_lower_text",
-          "i18n_template" => "higher_or_lower",
-        },
-      ],
-    })
-  end
```

Comment:
did this need updating instead of removing?

Reply:
Yes i've updated it